openfisca_france/parameters/prestations/prestations_familiales/af/modulation/plafond_tranche_1.yaml

```
@@ -12,6 +12,6 @@ values:
   2019-01-01:
     value: 68217.0
     reference: https://www.legifrance.gouv.fr/eli/arrete/2018/12/21/SSAS1835267A/jo/article_1 - 56849 + 5684 * 2
-  2020-01-01:
+  2020-04-01:
     value: 69309.0
-    reference: https://solidarites-sante.gouv.fr/fichiers/bo/2019/19-12/ste_20190012_0000_0064.pdf
+    reference: https://www.legifrance.gouv.fr/eli/arrete/2018/12/21/SSAS1835267A/jo/article_1 - 57759 + 5775 * 2
```

@benjello L'article 1 de la référence suivante me semble effectivement en faveur du 1er janvier 2020. 
cc @mtifarine @frtomas 
Suggestion de correction (+ conservation du 1er janvier) : 
```suggestion
    reference: https://www.legifrance.gouv.fr/eli/arrete/2019/12/20/SSAS1934813A/jo/texte
```